manifests/s/Shure/UpdateUtility/2.7.0/Shure.UpdateUtility.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Shure.UpdateUtility
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: Shure
+PublisherUrl: https://www.shure.com/en-US/products/software/shure_update_utility
+PublisherSupportUrl: https://www.shure.com/en-US/products/software/shure_update_utility#support
+PrivacyUrl: https://www.shure.com/en-US/meta/privacy
+Author: Shure
+PackageName: Update Utility
```

```suggestion
PackageName: Shure Update Utility
```
This must match Control Panel.